README.md

```diff
@@ -122,3 +157,38 @@ encode_number_precision
 This fork allows encoding of numbers with a `precision` up to 16 decimals (vs. 14 in mpx/lua-cjson).
 
 [Back to TOC](#table-of-contents)
+
+decode_array_with_array_mt
+--------------------------------
+**syntax:** `cjson.decode_array_with_array_mt(enabled)`
+
+**default:** true
+
+If enabled, JSON Arrays decoded by `cjson.decode` will result in Lua
+tables with the [`array_mt`](#array_mt) metatable. This can ensure a 1-to-1
+relationship between arrays upon multiple encoding/decoding of your
+JSON data with this module.
+
+If disabled, JSON Arrays will be decoded to plain Lua tables, without
+the `array_mt` metatable.
+
+The `enabled` argument can be a boolean, or a string (`on` or `off`).
```

Comment:
I think we should only support boolean values here, at least in the documentation.

Reply:
We refer to the `"on"` and `"off"` variants of the arguments in the documentation for [encode_empty_table_as_object()](https://github.com/openresty/lua-cjson#encode_empty_table_as_object) already. Should we remove such mentions from there as well?

Reply:
@thibaultcha Yes, please.